cmd/govim/complete.go

```diff
@@ -61,6 +61,35 @@ func (v *vimstate) complete(args ...json.RawMessage) (interface{}, error) {
 	}
 }
 
+func (v *vimstate) completeDone(args ...json.RawMessage) error {
+	currBufNr := v.ParseInt(args[0])
+	b, ok := v.buffers[currBufNr]
+	if !ok {
+		return fmt.Errorf("failed to resolve buffer %v", currBufNr)
+	}
+	var chosen struct {
+		Word string `json:"word"`
+		Abbr string `json:"abbr"`
+	}
+	v.Parse(args[1], &chosen)
+	if chosen.Word == "" {
+		return nil
+	}
+	var match *protocol.CompletionItem
+	for _, c := range v.lastCompleteResults.Items {
```

Comment:
This will panic with a nil-deref if the user performs a content-based completion (C-x C-n) without performing a omnicomplete before.

Reply:
Resolved per previous point.